src/continuum_opacity/hydrogenic_bf_ff.jl

```diff
@@ -0,0 +1,249 @@
+                                             ν, ρ, T, ion_energy)
+
+    κ_low_n + (κ_high_n * integrate_high_n)
+end
+
+
+
+# this table is taken from section 5.1 of Kurucz (1970)
+_ff_table_val = [5.53 5.49 5.46 5.43 5.40 5.25 5.00 4.69 4.48 4.16 3.85;
+                 4.91 4.87 4.84 4.80 4.77 4.63 4.40 4.13 3.87 3.52 3.27;
+                 4.29 4.25 4.22 4.18 4.15 4.02 3.80 3.57 3.27 2.98 2.70;
+                 3.64 3.61 3.59 3.56 3.54 3.41 3.22 2.97 2.70 2.45 2.20;
+                 3.00 2.98 2.97 2.95 2.94 2.81 2.65 2.44 2.21 2.01 1.81;
+                 2.41 2.41 2.41 2.41 2.41 2.32 2.19 2.02 1.84 1.67 1.50;
+                 1.87 1.89 1.91 1.93 1.95 1.90 1.80 1.68 1.52 1.41 1.30;
+                 1.33 1.39 1.44 1.49 1.55 1.56 1.51 1.42 1.33 1.25 1.17;
+                 0.90 0.95 1.00 1.08 1.17 1.30 1.32 1.30 1.20 1.15 1.11;
+                 0.45 0.48 0.52 0.60 0.75 0.91 1.15 1.18 1.15 1.11 1.08;
+                 0.33 0.36 0.39 0.46 0.59 0.76 0.97 1.09 1.13 1.10 1.08;
+                 0.19 0.21 0.24 0.28 0.38 0.53 0.76 0.96 1.08 1.09 1.09]
+# The x-axis is log₁₀(RydbergH*Z²/(k*T))
+_ff_xaxis = [-3.0, -2.5, -2.0, -1.5, -1.0, -0.5,  0.0,  0.5,  1.0,  1.5,  2.0]
+# The y-axis is log₁₀(h*ν/(k*T))
+_ff_yaxis = [-4.0, -3.5, -3.0, -2.5, -2.0, -1.5, -1.0, -0.5,  0.0,  0.5,  1.0,  1.5]
+
+# Since the grid is regularly spaced, we could probably make this faster
+# pass y,x to the interpolator
+_ff_interpolator = LinearInterpolation((_ff_yaxis, _ff_xaxis), _ff_table_val,
+                                       extrapolation_bc=Throw())
+
+"""
+    hydrogenic_ff_opacity(Z, ni, ne, ν, T)
+
+computes the free-free opacity for a hydrogenic species
+
+# Arguments
+- `Z::Integer`: the charge of the ion. For example, this is 1 for ionized H.
+- `ni::Flt`: the number density of the ion species.
+- `ne::Flt`: the number density of free electrons.
+- `ν::Flt`: frequency in Hz
+- `T::Flt`: temperature in K
+
+# Note
+This approach was adopted from section 5.1 of Kurucz (1970). The table is old, and there might be
+better data available. The table's x-axis is log₁₀(RydbergH*Z²/(k*T)) and the y-axis is 
+log₁₀(h*ν/(k*T)). There some ambiguity over whether we should replace RydbergH*Z² with the 
+ionization energy, but it's probably a negligible difference (given the log scale).
+
+An alternative approach is to directly compute the free-free opacity. According to equation (5.18b)
+of Rybicki & Lightman (2004), the opacity is given by 
+    3.7e8 * Z * Z *ne * ni * (1 - exp(-hplanck*ν/(kboltz*T))) * g / (sqrt(T) * ν * ν * ν)
+where g is the gaunt factor (a more exact coefficient can be computed from eqn 5.18a). Under this
+alternative approach, we might choose to interpolate the tabulated gaunt factors reported by
+van Hoof, Ferland, Williams, Volk, Chatzikos, Lykins, & Porter (2013, 2015).
+"""
+function hydrogenic_ff_opacity(Z::Integer, ni::Flt, ne::Flt, ν::Flt,
+                               T::Flt) where {Flt<:AbstractFloat}
+    β = 1.0/(kboltz_eV * T)
```

Comment:
Yeah, I agree.

For stellar spectroscopy, it may be more common notation to define `θ = log10(e)/(kB*T) ~ 5040/T` and evaluate `10^(energy*θ)`. This is equivalent to `exp(energy*β)`. But I think it would be easier to vectorize `exp(energy*β)` (plus the notation is probably more familiar to anyone who has ever seen statistical mechanics)